tests/markdown/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`markdown.md 1`] = `
+\`\`\`php
+$foo = 'bar';
+
+\`\`\`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+\`\`\`php
+$foo = 'bar';
+\`\`\`
+
```

Comment:
I think we should avoid newline in end in markdown

Reply:
I think we need to though, or else it will break the markdown? Unless you know of a another way to fix #423?

Reply:
@mgrip I looked wrong place, all right here